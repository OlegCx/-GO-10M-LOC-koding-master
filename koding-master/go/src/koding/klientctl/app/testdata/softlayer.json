{
  "provider": {
    "softlayer": {
      "api_key": "${var.softlayer_api_key}",
      "username": "${var.softlayer_username}"
    }
  },
  "resource": {
    "softlayer_virtual_guest": {
      "softlayer-instance": {
        "cpu": "1",
        "domain": "koding.com",
        "hourly_billing": true,
        "image": "UBUNTU_14_64",
        "local_disk": true,
        "name": "softlayer-instance",
        "public_network_speed": 10,
        "ram": "1024",
        "region": "dal09",
        "user_data": "\necho \"hello world!\" >> /helloworld.txt\n"
      }
    }
  }
}
