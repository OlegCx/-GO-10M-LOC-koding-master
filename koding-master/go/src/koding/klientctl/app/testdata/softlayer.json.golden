{
  "provider": {
    "softlayer": {
      "api_key": "${var.softlayer_api_key}",
      "username": "${var.softlayer_username}"
    }
  },
  "resource": {
    "softlayer_virtual_guest": {
      "softlayer-instance": {
        "cpu": "1",
        "domain": "koding.com",
        "hourly_billing": true,
        "image": "UBUNTU_14_64",
        "koding_always_on": true,
        "local_disk": true,
        "name": "softlayer-instance",
        "public_network_speed": 10,
        "ram": "1024",
        "region": "dal09",
        "user_data": "#cloud-config\nruncmd:\n- echo \"hello world!\" >> /helloworld.txt\nssh_authorized_keys:\n- ssh-rsa AAAAB3NzaC1yc2EA... koding-350298856\n"
      }
    }
  }
}
